ethereum/api/src/test/java/org/hyperledger/besu/ethereum/api/jsonrpc/internal/parameters/FilterParameterTest.java

```diff
@@ -224,6 +238,83 @@ public void emptyListAsSubTopicDecodesCorrectly() throws JsonProcessingException
             emptyList());
   }
 
+  @Test
+  public void emptyListOfTopicsInJsonGivesInEmptyListInFilterParameter()
+      throws java.io.IOException {
+    final FilterParameter filterParameter = readJsonAsFilterParameter("{\"topics\":[]}");
+
+    assertThat(filterParameter.getTopics().size()).isEqualTo(0);
+  }
+
+  @Test
+  public void emptyListOfTopicsNestedGivesEmptyListInTopics() throws java.io.IOException {
```

Comment:
Maybe `nestedEmptyTopicsListShouldDeserializeAsEmptyList()`?

(also it is more usual for unqualified exceptions e.g. `IOException`)